ada/language/ast.py

```diff
@@ -5449,7 +5451,7 @@ class FormalSubpDecl(ClassicSubpDecl):
     default_expr = Field(type=T.Expr)
     aspects = Field(type=T.AspectSpec)
 
-    defining_names = Property(Self.subp_spec.name.as_entity.singleton)
+    defining_names = Property(Entity.subp_spec.name.as_entity.singleton)
```

Comment:
Could `.as_entity` be removed? If it can, I wonder why Langkit does not compilain that `.as_entity` input is already an entity… (I’ll have a look)